docs/pages/setup/guides/joining-nodes-aws-ec2.mdx

```diff
@@ -207,6 +211,16 @@ proxy_service:
 
 ## Step 4/4. Launch your Teleport node
 
+<Admonition
+  type="note"
+  title="IMPORTANT"
+>
```

Comment:
I think this tag in unclosed

Reply:
Thanks for reviewing, it was a lint issue.  There is a tag to close this.